DependencyInjection/Configuration/Configuration.php

```diff
@@ -135,7 +135,8 @@ private function addClientsSection(ArrayNodeDefinition $rootNode)
                                     ->end()
                                     ->scalarNode('cluster')->defaultNull()->end()
                                     ->scalarNode('prefix')->defaultNull()->end()
-                                    ->booleanNode('replication')->defaultFalse()->end()
+                                    ->enumNode('replication')->values(array(true, false, 'sentinel'))->defaultNull()->end()
+                                    ->scalarNode('service')->defaultNull()->end()
```

Comment:
Adding this will enable people to select a database if they choose:
` ->arrayNode('parameters')
      ->children()
         ->scalarNode('database')->defaultNull()->end()
       ->end()
 ->end()`